packages/react-make-styles/bundle-size/makeStyles.fixture.js

```diff
@@ -0,0 +1,7 @@
+import { __styles, mergeClasses } from '@fluentui/react-make-styles';
```

Comment:
**q:** is the structure hard coded ? meaning: to be able to make it work, fixtures have to live under `<package-name>/bundle-size/**` ?

Reply:
based on code - it is hardcoded

Reply:
Yes, it's hardcoded